README.md

```diff
@@ -65,17 +65,12 @@ function oidc_clients() {
 }
 ```
 
-## Development
-Start local site with:
+## Override templates
+You can override templates provided by this plugin from a theme. To do so, you should create an `openid-connect/` directory under your theme's root:
 
 ```shell
-wp-env start
+wp-content/themes/my-theme/
+ │── openid-connect/
+ │──── main.php
+ └──── forbidden.php
 ```
```

Comment:
I think it would be great to also link the Github for that class we are using here, so that folks can learn more about it & a heads up for those familiar with it already.

Reply:
I think linking to the docs of [Gamajo-Template-Loader](https://github.com/GaryJones/Gamajo-Template-Loader) could be confusing to our users, who could be non-technical or "half-technical", as those docs are more targeted at plugin developers, and contain a lot of information that is not that relevant when you just want to override a template.

Additionally, the fact that we're using Gamajo-Template-Loader is a detail of the current implementation, in the future we could move away from using it. Linking to their docs might give the impression to our users that Gamajo-Template-Loader is part of our public API.

I think it's best that consumers of this plugin only interact with the templating system through `Templating.php`, as that gives us the possibility of changing the internals while maintaining compatibility, as long as we don't change its public API.

I improved the docs on overriding templates so that it's more clear how to do it. Does this make sense @ashfame? Do you think the current docs are clear and enough for someone who wants to override a template?

Reply:
Ok, that makes sense! Yes, the instructions are quite detailed :+1: 

I was only thinking about mentioning support for child/parent theme when I made that comment, as in, order of overriding templates `childtheme > parenttheme > plugin`.

Reply:
Makes sense. I pushed a commit to mention parent and child themes.